### ISSUE TITLE
package actively maintained?

### ISSUE THREAD

**Issue by mmahalwy** [final-form/react-final-form#955]
Curious if the package plans to be maintained in the future.

**Comment by kennybecerra** [comment]
Bumping this up as well. We are currently in the process of migrating away from Redux-Form. React-Final-Form seemed like a logical step forward but I am concerned on the lack of response and timeline on some of the issues in the repo.

**Comment by mmahalwy** [comment]
@erikras bump? :)

**Comment by tnrich** [comment]
@erikras unfortunately seems to have a bit of a history of abandoning very popular OSS he's made..

**Comment by msageryd** [comment]
Hmm, not good. I'm deeply into Final-form after evaluating FF against Formik. I might have to revisit Formik in order to secure the future of my app.. Unfortunately, FF suits my need much better that Formik, so I'd rather not switch.

@erikras Have you thought about taking in other maintainers? I would'nt qualify for this, but if some serious maintainers would get in I could help out with testing and docs. I don't think any OSS can survive without having at least a couple of maintainers.

**Comment by erikras** [comment]
Yes, I'd be open to having someone help maintain the package. DM me on Twitter if you're interested. @erikras 

Sorry for the delayed response. GH notifications are too noisy.

**Comment by msageryd** [comment]
I fully understand that life can interfere with OSS work. Unfortunately, this came to late for me. I've migrated to another form handler as I worried about continuation of FF.

I hope that someone steps up and helps out with this. Even though I've moved on, I really liked FF.

**Comment by smac89** [comment]
This issue should probably be pinned so that casual readers will see it, but more importantly to attract a new maintainer
